CMakeLists.txt

```diff
@@ -205,6 +206,12 @@ else()
   target_compile_definitions(snmalloc INTERFACE SNMALLOC_USE_WAIT_ON_ADDRESS=0)
 endif()
 
+if(SNMALLOC_USE_SELF_VENDORED_STL)
+  target_compile_definitions(snmalloc INTERFACE SNMALLOC_USE_SELF_VENDORED_STL=1)
+else()
+  target_compile_definitions(snmalloc INTERFACE SNMALLOC_USE_SELF_VENDORED_STL=0)
+endif()
+
```

Comment:
I've noticed as a style you tend to use 1/0 to signify a boolean, but you can do his with defined and not defined.  The advantage of defined and not defined is that it doesn't require the build to setup as many defines as it will default to not defined.

In this instance, not self vendoring seems like a good idea.  If you are using this for allowing multiple self vendorings, then we could do undefined as not, and define it as "gcc" or other things for the relevant directories.

Reply:
Good idea. Changing now.